python/ray/train/xgboost/config.py

```diff
@@ -67,12 +136,14 @@ def on_training_start(
         # NOTE: sortby="task" is needed to ensure that the xgboost worker ranks
         # align with Ray Train worker ranks.
         # The worker ranks will be sorted by `DMLC_TASK_ID`,
-        # which is defined in `on_training_start`.
+        # which is defined below.
         self._tracker = RabitTracker(
-            host_ip=train_driver_ip, n_workers=num_workers, sortby="task"
+            n_workers=num_workers, host_ip=train_driver_ip, sortby="task"
         )
-        rabit_args.update(self._tracker.worker_envs())
-        self._tracker.start(num_workers)
+        self._tracker.start(n_workers=num_workers)
+
+        worker_args = self._tracker.worker_envs()
+        rabit_args.update(worker_args)
```

Comment:
Is the worker_args contains all the env vars of a worker? Or only those env vars related to Rabit?

Reply:
just env vars related to rabit initialization.